src/systems/joint_position_controller/JointPositionController.hh

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) 2020 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+#ifndef IGNITION_GAZEBO_SYSTEMS_JOINTPOSITIONCONTROLLER_HH_
+#define IGNITION_GAZEBO_SYSTEMS_JOINTPOSITIONCONTROLLER_HH_
+
+#include <ignition/gazebo/System.hh>
+#include <memory>
+
+namespace ignition
+{
+namespace gazebo
+{
+// Inline bracket to help doxygen filtering.
+inline namespace IGNITION_GAZEBO_VERSION_NAMESPACE {
+namespace systems
+{
+  // Forward declaration
+  class JointPositionControllerPrivate;
+
+  /// \brief Joint position controller which can be attached to a model with a
+  /// reference to a single joint.
+  //  Currently only the first axis of a joint is actuated.
```

Comment:
Now we can choose the joint index :wink: 

We've been trying to add usage documentation to each system's header. Do you mind documenting the SDF parameters and transport interface here? See an example [here](https://github.com/ignitionrobotics/ign-gazebo/blob/master/src/systems/triggered_publisher/TriggeredPublisher.hh).

Reply:
> Now we can choose the joint index 
> 
> We've been trying to add usage documentation to each system's header. Do you mind documenting the SDF parameters and transport interface here? See an example [here](https://github.com/ignitionrobotics/ign-gazebo/blob/master/src/systems/triggered_publisher/TriggeredPublisher.hh).

A good thing to do :) a7ad6ef Thanks!